### PR TITLE
(Update): Regenerated OpenAPI code

### PR DESCRIPTION
**Pull request by johnorcutt**
Regenerated OpenAPI code for latest API changes (annual temp, precipitation). 
Removed UBD and LBD helpers

### CODE REVIEW

File: clients/python/sust/api/generated/climate_explorer/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 """
     Sust Global Climate Explorer API
 
-     This API provides programmatic access to physical risk exposure data. For more guidance on using this API, please visit the Sust Global Dev Center: https://developers.sustglobal.com.   # noqa: E501
+    This API provides programmatic access to physical risk exposure data.For more guidance on using this API, please visit the Sust Global Dev Center: https://developers.sustglobal.com.  # noqa: E501
```
Comment (bcwaldon) [inline clients/python/sust/api/generated/climate_explorer/__init__.py:6]: Lost one important space in there : `data.For`

Comment (bcwaldon) [reply clients/python/sust/api/generated/climate_explorer/__init__.py:6]: Did the openapi generator do this?

**Review by bcwaldon — CHANGES_REQUESTED** [review]
Let's figure out why there are so many whitespace changes here before merging.

File: clients/python/sust/api/generated/climate_explorer/__init__.py (hunk 1 above)
Comment (johnorcutt) [reply clients/python/sust/api/generated/climate_explorer/__init__.py:6]: Yeah, the generator did this. I didn't touch any of those files.

**Pull request merged**